### PR TITLE
DHFPROD-7828: Component Replacement: PageHeader

### PR DESCRIPTION
**Pull request by yubiselmarklogic**
#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Code passes ESLint tests
- [x] Added Tests
- [x] Run UI tests
  

- ##### Reviewer:

- [x] Reviewed Tests

### CODE REVIEW

**Comment by federicoRufrancosMarklogic** [comment]
Some details I found comparing both:
* Increase font weight
* Decrease padding top
Original one
image
Bootstrap replacement
image

**Review by federicoRufrancosMarklogic — COMMENTED** [review]
Code looks good, just take a look to the previous comparison and see if you could solve that

**Comment by yubiselmarklogic** [comment]
@federicoRufrancosMarklogic I fixed the comments

**Review by federicoRufrancosMarklogic — APPROVED** [review]

File: marklogic-data-hub-central/ui/src/HCTheme.scss [hunk 1]
```diff
@@ -150,4 +150,8 @@ $input-border-radius: 2px;
 
 .btn-sm, .btn-group-sm > .btn {
     padding: 0.05rem 0.45rem;
-  }
+}
+
+.cursor-pointer{
```
Comment (mrodrigu-marklogic) [inline marklogic-data-hub-central/ui/src/HCTheme.scss:155]: Please add a space between the class name and the bracket here

**Review by mrodrigu-marklogic — CHANGES_REQUESTED** [review]

**Review by federicoRufrancosMarklogic — APPROVED** [review]

File: marklogic-data-hub-central/ui/src/HCTheme.scss (hunk 1 above)
Comment (yubiselmarklogic) [reply marklogic-data-hub-central/ui/src/HCTheme.scss]: @mrodrigu-marklogic done

**Review by federicoRufrancosMarklogic — APPROVED** [review]

**Review by wooldridge — APPROVED** [review]
Looks good!
Screen Shot 2021-09-29 at 1 56 11 PM

**Review by iveyMarklogic — APPROVED** [review]

**Pull request merged**